webvtt.go

```diff
@@ -37,6 +39,11 @@ var (
 	webVTTUnescaper                    = strings.NewReplacer("&amp;", "&", "&lt;", "<")
 )
 
+// endOfCss returns true if the cssLines are empty or the last line is the end token
+func endOfCss(cssLines []string) bool {
```

Comment:
It feels weird to create a function here since it's only use once below and condition is pretty simple  🤔

Reply:
The function will make the condition shorter, if we do not use this function, the condition will become:
```
if blockName != webvttBlockNameStyle || webVTTStyles == nil || len(webVTTStyles.WebVTTStyles) > 0 || webVTTStyles.WebVTTStyles[len(webVTTStyles.WebVTTStyles)-1] == cssEndToken 
```
very long and uneasy to read,

Reply:
I refer the shorten style with function since the function name state it clear what we are checking. But I am not against the long condition, so I will change the condition then.